tests/test_pigeon.py

```diff
@@ -330,7 +343,7 @@ def test_modify_metadata_not_public(self, mock_ia_client, guid):
             "title": "Test Component",
             "description": "Test Description",
             "date": "2017-12-20",
-            "moderation_state": "withdrawn",
```

Comment:
These values changed while the tests were offline, so they are adjusted now that they are back.